packages/oid4vci-demo-frontend/src/pages/SSICredentialVerifyRequestPage/index.tsx

```diff
@@ -119,7 +115,7 @@ export default function SSICredentialVerifyRequestPage(): React.ReactElement | n
                             <SSIPrimaryButton
                                 caption={t('credential_verify_request_right_pane_button_caption')}
                                 onClick={async () => {
-                                    sequencer.goToStep(config.rightPaneButtonStepId ?? 'infoRequest')
+                                    flowRouter.goToStep(config.rightPaneButtonStepId ?? 'infoRequest')
```

Comment:
@sksadjad is there a point to add await if you don't have any follow-up code that has to wait for it?

Reply:
in general I think it's better to wait for your async function. it can cause resource issues, it can wait for it indefinitely or other problems/issues

Reply:
fixed